### DISCUSSION TITLE
安卓端1.8.18  内核1.8.9，开启代理，允许局域网连接后，只有telegram应用无法联网，换了节点也不行

### DISCUSSION THREAD

**Discussion by pp6451303** [2dust/v2rayNG#3092] [General]
在提出问题前请先自行排除服务器端问题，同时也请通过搜索确认是否有人提出过相同问题。

### 预期行为
描述你认为应该发生什么

### 实际行为
描述实际发生了什么

### 复现方法
1.
2.
3.

### 日志信息
<details>

通过`adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main`获取日志。请自行删减日志中可能出现的敏感信息。

如果问题可重现，建议先执行`adb logcat -c`清空系统日志再执行上述命令，再操作重现问题。
```
在这里粘贴日志
```
</details>

**Comment by ldd200888** [comment]
我也遇到了这个问题

**Reply by ldd200888** [reply]
浏览器中https://telegram.org/是可以访问，但是就是电报里面无法连接

**Reply by Y-D-Angle** [reply]
已解决，下载猫报

**Comment by 502431670** [comment]
最近V2rayNG 1.8.25 版本，不显示VPN代理图标，多次清空数据，以后反复卸载和安装，安装好第一次可以显示VPN代理图标，关闭V2软件之后再次开启代理手机顶部就不显示VPN代理图标了，是什么原因导致？还是说V2rayNG有一个隐藏设置可以把VPN代理图标隐藏？需要要在V2上面怎么设置才可以吧VPN图标显示出来？

**Comment by 502431670** [comment]
最近V2rayNG 1.8.25 版本，不显示VPN代理图标，多次清空数据，以后反复卸载和安装，安装好第一次可以显示VPN代理图标，关闭V2软件之后再次开启代理手机顶部就不显示VPN代理图标了，是什么原因导致？还是说V2rayNG有一个隐藏设置可以把VPN代理图标隐藏？需要要在V2上面怎么设置才可以吧VPN图标显示出来？

**Comment by EluneAdore** [comment]
同样的问题就电报无法使用

**Comment by CoderLiuChina** [comment]
同样的问题，有人解决了吗
